es/criteria/bundle-policy-and-code.md

```diff
@@ -2,49 +2,49 @@
 order: 2
 ---
 
-#Política de paquetes y código fuente
+# Política de paquetes y código fuente
 
 ## Requisitos
 
 * Una codebase DEBE incluir la política en la que se basa el código fuente.
 * Una codebase DEBE incluir todo el código fuente en el que se basa la política.
 * Toda la política y el código fuente en el que se basa la codebase DEBEN estar documentados y ser reutilizables y portables.
-* La política DEBERÍA ser proporcionada en formatos legibles por la máquina y sin ambigüedades.
+* La política DEBERÍA ser proporcionada en formatos sin ambigüedades y legibles por computador.
 * Las pruebas de integración continua DEBERÍAN validar que el código fuente y la política se ejecutan de forma coherente.
 
 ## Por qué es importante
 
 Así se garantiza el acceso tanto al código fuente como a los documentos normativos para facilitar la reutilización efectiva de una codebase.
 
-## WQué no hace
+## Qué no hace
 
 * Garantizar que una codebase refleje la política del paquete.
 * Asegurarse de que los paquetes cumplen con la infraestructura técnica local o el marco legal de una determinada organización pública.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks.